src/main/java/core/basesyntax/Menu.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Scanner;
+
+public class Menu {
```

Comment:
move all the logic to different class. From main method your should call only one method which will run your application